src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
```

Comment:
I usually solve this problem by putting the content into a large drawing where I can then move the text and formulas around. It is not optimal but what you suggest is a *major* change in TeXmacs. Actually is not even very easy to do in LaTeX if I'm not wrong.

Reply:
Indeed. The frustrating point is that it is possible in virtually all wysiwyg text editors (LibreOffice, MS Office, Pages...) and that I need to do that in nearly every page of most document I type.
The fact that it is a major change is why I put this wish here in the first place, and not on the bug list.